### DISCUSSION TITLE
Exclude page from sitemap but not from collections

### DISCUSSION THREAD

**Discussion by benpages** [11ty/eleventy#2383] [Q&A] [answered]
Hey there,

On my site [under Portfolio](https://benpages.com/portfolio/), I have a page listing 24 external links.

I'm using the following loop on the portfolio page: 
`{%- for post in collections.article | reverse -%}
      {% include 'article-list.njk' %}
    {%- endfor -%}`

Then, I have a folder in my source directory called articles. For each article on the portfolio page, I have a file with front-matter only. It includes the image path, title, date, etc. It also includes the target URL, which is the external URL I want visitors to be redirected to when clicking on one of these posts. 

This works well as adding new articles to that page is very quick and easy. No need to toy with HTML.

However, in my public folder, Eleventy creates an individual page for each of the articles in the articles folder. So I have a bunch of empty pages on my site: /articles/slug/index.html. I would like to at least be able to remove these from the sitemap so they don't get indexed.

I tried using eleventyExcludeFromCollections: true but the article is then no longer indexed on my Portfolio page. 

Is there a workaround or did I dig myself into a hole here?

**Comment by benpages** [comment, marked as answer]
Found the solution. Inject `permalink: false` into the front matter of each article.

**Reply by zachleat** [reply]
Yeah! I would also note that you can set `permalink` using a Directory data file too, if you want it to apply to all of the files in the folder: https://www.11ty.dev/docs/data-template-dir/
